vertical-pod-autoscaler/pkg/recommender/main.go

```diff
@@ -122,14 +127,30 @@ func main() {
 	if *postProcessorCPUasInteger {
 		postProcessors = append(postProcessors, &routines.IntegerCPUPostProcessor{})
 	}
+
 	// CappingPostProcessor, should always come in the last position for post-processing
 	postProcessors = append(postProcessors, &routines.CappingPostProcessor{})
+	var source input_metrics.PodMetricsLister
+	if *useExternalMetrics {
+		resourceMetrics := map[apiv1.ResourceName]string{}
+		if externalCpuMetric != nil && *externalCpuMetric != "" {
+			resourceMetrics[apiv1.ResourceCPU] = *externalCpuMetric
+		}
+		if externalMemoryMetric != nil && *externalMemoryMetric != "" {
+			resourceMetrics[apiv1.ResourceMemory] = *externalMemoryMetric
+		}
+		externalClientOptions := &input_metrics.ExternalClientOptions{ResourceMetrics: resourceMetrics, ContainerNameLabel: *ctrNameLabel}
+		klog.V(1).Infof("Using External Metrics: %+v", externalClientOptions)
+		source = input_metrics.NewExternalClient(config, clusterState, *externalClientOptions)
+	} else {
+		source = input_metrics.NewPodMetricsesSource(resourceclient.NewForConfigOrDie(config))
```

Comment:
Please log here, similar to what we log in line 143; so we don't have to guess that we're using metrics from metrics server by absence of log about using external metrics.

Reply:
Done.